src/Tribe/Repository/Query_Filters.php

```diff
@@ -616,16 +616,6 @@ protected function where_field_is( $where, WP_Query $query, $field, $prepare = '
 		return $where;
 	}
 
-	/**
```

Comment:
Just a note - this is being removed because during shutdown it slows down due to garbage collection(?), and it is not likely necessary (don't think destruct would run til shutdown anyways, there are object references in the filter list...).